### ISSUE TITLE
Automated builds

### ISSUE THREAD

**Issue by deurk** [deurk/ktx#3]
Automated builds should be generated for each change and made available for download.

**Comment by deurk** [comment]
Discussions have restarted around using Travis to automate builds. Help is welcome regarding the setup of those.

**Comment by nvtkaszpir** [comment]
I've created  PR with travis + meson builds - it should be a simple starting point for improvements.

Meson used for cross compiled code for linux 32/64bit, windows 32/64bit , arm (raspberry pi 3 with raspbian anyone?). I added short section to readme how to do it. I can provide vagrant or maybe even docker builds if anyone interested, but I guess empezar is doing something with containers.

Actually did not have time to test if the binaries work - tried with rpi3 but it whined that it needs new api version, and deurk/mvdsv/master is just not compatible yet.

Also storing artifacts is needed and some more sane tests :D
